src/Estimators/src/ZeroVelocityDynamics.cpp

```diff
@@ -0,0 +1,167 @@
+/**
+ * @file ZeroVelocityDynamics.cpp
+ * @authors Ines Sorrentino
+ * @copyright 2023 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the BSD-3-Clause license.
+ */
+
+#include <BipedalLocomotion/TextLogging/Logger.h>
+#include <BipedalLocomotion/RobotDynamicsEstimator/ZeroVelocityDynamics.h>
+
+namespace RDE = BipedalLocomotion::Estimators::RobotDynamicsEstimator;
+
+bool RDE::ZeroVelocityDynamics::initialize(std::weak_ptr<const ParametersHandler::IParametersHandler> paramHandler)
+{
+    constexpr auto errorPrefix = "[ZeroVelocityDynamics::initialize]";
+
+    auto ptr = paramHandler.lock();
+    if (ptr == nullptr)
+    {
+        log()->error("{} The parameter handler is not valid.", errorPrefix);
+        return false;
+    }
+
+    // Set the state dynamics name
+    if (!ptr->getParameter("name", m_name))
+    {
+        log()->error("{} Error while retrieving the name variable.", errorPrefix);
+        return false;
+    }
+
+    // Set the state process covariance
+    if (!ptr->getParameter("covariance", m_covariances))
+    {
+        log()->error("{} Error while retrieving the covariance variable.", errorPrefix);
+        return false;
+    }
+
+    // Set the state initial covariance
+    if (!ptr->getParameter("initial_covariance", m_initialCovariances))
+    {
+        log()->debug("{} Variable initial_covariance not found.", errorPrefix);
```

Comment:
I decided to use two different classes instead of one. Indeed, the meaning of the two models for the state and for the measurement is different even if they implement the same equation.
The other difference is that for the process the initial covariance is required instead for the measurements it is not. Further, the user can decide to use a bias in the measurement model which is not allowed for the state.

Given that, having two different classes is much more intuitive.